index.php

```diff
@@ -2,26 +2,34 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    
+<?php
 
-    <title>Template</title>
+use Collection\Monitors\Models;
 
-    <meta name="description" content="Template HTML file">
-    <meta name="author" content="iO Academy">
+require_once 'vendor/autoload.php';
 
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
-    <link rel="stylesheet" href="css/styles.css">
+// Getting connected
+$db = new PDO('mysql:host=db; dbname=collection', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
 
-    <link rel="icon" href="images/favicon.png" sizes="192x192">
-    <link rel="shortcut icon" href="images/favicon.png">
-    <link rel="apple-touch-icon" href="images/favicon.png">
+$model = new Models($db);
 
-    <script defer src="js/index.js"></script>
-</head>
+$monitors = ($model->getAllProducts());
```

Comment:
You have an extra set of brackets around `$model->getAllProducts()` that you don't need

Reply:
Not really sure why I had them but I think I tried a different way beforehand and didn't remove